source/menu_options.c

```diff
@@ -32,7 +33,7 @@ static void _draw_OptionsMenu(u8 alpha)
 				DrawTexture(&menu_textures[c], OPTION_ITEM_OFF - 29, y_off, 0, menu_textures[c].width, menu_textures[c].height, 0xFFFFFF00 | alpha);
 				break;
 			case APP_OPTION_CALL:
-				DrawTexture(&menu_textures[footer_ico_cross_png_index], OPTION_ITEM_OFF - 29, y_off+2, 0, menu_textures[footer_ico_cross_png_index].width, menu_textures[footer_ico_cross_png_index].height, 0xFFFFFF00 | alpha);
+				DrawTexture(&menu_textures[orbisPadGetConf()->crossButtonOK ? footer_ico_cross_png_index : footer_ico_circle_png_index], OPTION_ITEM_OFF - 29, y_off+2, 0, menu_textures[footer_ico_cross_png_index].width/2, menu_textures[footer_ico_cross_png_index].height/2, 0xFFFFFF00 | alpha);
```

Comment:
why reducing the icon size by half? (`width/2`)

Reply:
Copied it from cheat manager

Reply:
I didn't even realize it was here https://github.com/GoldHEN/GoldHEN_Cheat_Manager/blob/59e65c867c6d9edc82ebb58c7b46b0183ec2a30e/source/menu_options.c#L36

Reply:
oh alright, that's because assets in GH Cheat manager are different so sizes probably don't match.